### PR TITLE
XOR-550 [FIX] Remove duplicate message when we create new group with same email list

### PR DESCRIPTION
**Pull request by amolsuryawanshi4**
**Product areas affected**

chat component

**What does this PR do?**
In this PR we fixed below thing:
- [ ] if the conversion is already exist in group, so we removed duplicate message when we create new group with same email list.

**JIRA ticket**

[click here](https://weboo.atlassian.net/browse/XOR-550)

**Result**

image

**Checklist**

None

**Testing instructions**

None

**Deployment instructions**

None

**Author concerns**

None

### CODE REVIEW

**Review by yogeshmsharma-xoriant — APPROVED** [review]
Please add detailed comment

File: js/build.js [hunk 1]
```diff
@@ -2426,7 +2426,7 @@ Fliplet().then(function() {
       }
     }
 
-    function viewConversation(conversation) {
+    function viewConversation(conversation, isNewConversation = true) {
```
Comment (yogeshmsharma-xoriant) [inline js/build.js:2429]: rename this to isNew

**Review by yogeshmsharma-xoriant — APPROVED** [review]
make change and create PR For Fliplet

File: js/build.js (hunk 1 above)
Comment (amolsuryawanshi4) [reply js/build.js]: Done

**Review by NikitaJadhavXoriant — APPROVED** [review]
changes look ok to me

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**